src/NzbDrone.Core/DecisionEngine/DownloadDecisionComparer.cs

```diff
@@ -74,6 +83,31 @@ private int ComparePreferredWordScore(DownloadDecision x, DownloadDecision y)
             return CompareBy(x.RemoteEpisode, y.RemoteEpisode, remoteEpisode => remoteEpisode.PreferredWordScore);
         }
 
+        private int CompareIndexerPriority(DownloadDecision x, DownloadDecision y)
+        {
+            return CompareBy(x.RemoteEpisode, y.RemoteEpisode, GetIndexerPriority);
+        }
+
+        private int GetIndexerPriority(RemoteEpisode remoteEpisode)
+        {
+            if (remoteEpisode == null  || remoteEpisode.Release == null || remoteEpisode.Release.IndexerId == 0)
```

Comment:
We don't do these null checks anywhere else and things aren't blowing up, so these should be unnecessary.